README.md

```diff
@@ -2,26 +2,77 @@
 
 Prototype operator for [KubeFed](https://github.com/kubernetes-sigs/federation-v2). Planned to eventually replace the [federation-v2-operator repo](https://github.com/openshift/federation-v2-operator).
 
-Note, currently, this operator only supports deploying KubeFed in a namespace-scoped fashion.
+This operator supports deploying KubeFed in a namespace-scoped and a cluster-scoped fashion.
 
 ## Deploying and testing
 
 This work-in-progress section describes how people _developing_ this operator can deploy it.
 
+### Prerequisites
+- [Kind](https://github.com/kubernetes-sigs/kind) tool
+-  The latest version of [kubefedctl](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/userguide.md#kubefedctl-cli) binary installed in your $PATH 
+-  [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) for a kubernetes cluster
```

Comment:
Do you need `kind`, `minikube`, or both?

Reply:
I know the `Kind` tool will handle the first three use-cases for the smoke-test script i.e a local deployment, cluster, and olm-kube however, I want to segregate the olm-kube use-case from rest of the two so included `minikube` here explicitly. Let me know if you think otherwise.

Reply:
I don't think there's a reason to require both if either would do. Doesn't the olm-kube case work on kind?

Reply:
Yes, it works on Kind. Alright, as you suggested I will not add minikube in the prerequisites. Let me also make this change in the smoke-test script which is using minikube for the `olm-kube` use case.